rust/src/ike/mod.rs

```diff
@@ -14,11 +14,11 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
  * 02110-1301, USA.
  */
-
-// written by Pierre Chifflier  <chifflier@wzdftpd.net>
```

Comment:
Why did you remove Pierre ?

Reply:
Not intentionally, perhaps happened when creating the commits....